client/js/constants.js

```diff
@@ -56,6 +58,7 @@ const commands = [
 	"/os",
 	"/part",
 	"/query",
+	"/quote",
```

Comment:
As far as I remember, `/quote` being a simple alias of `/raw`, we didn't include it here on purpose. Shouldn't we remove `/quote` and `/send` here, and keep `/raw` only instead?

Reply:
Well, we have all the other aliases in this list, so it would be odd to not have 1 of the aliases of this particular one.